app/views/current/views/water/reference-number.html

```diff
@@ -12,10 +12,11 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-
-    <h1 class="govuk-heading-l">Reference number:</h1>
-    <h1 class="govuk-heading-l">GDA54X</h1>
-
+    {% from "govuk/components/panel/macro.njk" import govukPanel %}
```

Comment:
you didn't have to switch to this layout format - unless you are really comfortable with this coding pattern